frontend/src/components/widgets/DateInput/DateInput.tsx

```diff
@@ -181,7 +181,10 @@ class DateInput extends React.PureComponent<Props, State> {
     const { isEmpty } = this.state
     if (isEmpty) {
       this.setState(
-        { values: stringsToDates(this.props.element.default) },
```

Comment:
for some reason, `handleClose` called twice when calendar closed empty, which leads to two concurrent messages to the backend and breaks the widgets `old_state` `new_state` machinery. 

To prevent this we also sent `isEmpty` state during execution of handler, so on the second call `this.commitWidgetValue({ fromUi: true })` will not be called.